### PR TITLE
Update dlp-policy-reference.md

### PR DESCRIPTION
**Pull request by arunt14**
@chrfox , please consider this for approval.

Kindly ignore the PR sent earlier today

### CODE REVIEW

File: microsoft-365/compliance/dlp-policy-reference.md [hunk 1]
```diff
@@ -310,8 +310,8 @@ The available context options change depending on which location you choose. If
 ##### Conditions Devices supports
 
 - Content contains
-- (preview) Document or attachment is password protected (.pdf, Office files and Symantec PGP encrypted files are fully supported).This predicate doesn’t detect digital rights managed (DRM) encrypted or permission protected files.
-- (preview) Content is not labeled (.pdf, Office files are fully supported). This predicate detects content that doesn't have a sensitivity label applied. To help ensure only supported file types are detected, you should use this condition with the **File extension is** or **File type is** conditions.
+- Document or attachment is password protected (.pdf, Office files, .zip and Symantec PGP encrypted files are fully supported).This predicate detects only open protected files.
+- Content is not labeled (.pdf, Office files are fully supported). This predicate detects content that doesn't have a sensitivity label applied. To help ensure only supported file types are detected, you should use this condition with the **File extension is** or **File type is** conditions.
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/dlp-policy-reference.md]:
```suggestion
- Document or attachment is password protected (.pdf, Office files, .zip, and Symantec PGP encrypted files are fully supported). This predicate detects only open protected files.
- Content is not labeled (.pdf and Office files are fully supported). This predicate detects content that doesn't have a sensitivity label applied. To help ensure only supported file types are detected, you should use this condition with the **File extension is** or **File type is** conditions.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @arunt14, kindly commit the suggested changes. Could you please also close the earlier (duplicate) PR? Many thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by arunt14** [comment]
@chrfox , please confirm when this will be published

**Comment by arunt14** [comment]
@chrfox , can you pls confirm if the changes are pushed

**Pull request merged**